W-16112891 RPA Manager release notes 2024-06/3. (#394)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

RPA Manager is the cloud-based control plane for your automation projects.

[[june-27-2024]]
== June 27, 2024

// == What's New
//include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]

// == Fixed Issues
include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]

[%header,cols="6a,1a"]
|===
|Issue Resolution |ID

> | Now the user can configure the execution timetable. | W-16110013

Reviewer: ```suggestion
| You can now configure and save the execution timetable settings. | W-16110013
```

|===

// == Known Issues
include::partial$release-notes/rn-headings.adoc[tag=knownIssues,leveloffset=+2]

[%header,cols="3a,3a,1a"]
|===
|Known Issue |Workaround | ID

> | A new Anypoint Platform user is not available in dropdown lists. | The new Anypoint Platform user must log in at least once to RPA Manager to appear in the lists of available users. | W-13833054

Reviewer 2: ```suggestion
| A new Anypoint Platform user isn't available in dropdown lists. | The new Anypoint Platform user must log in at least once to RPA Manager to appear in the lists of available users. | W-13833054
```

> | A user with recently added RPA Project Manager permissions is not available in the dropdown menu for project manager assignments. | The user with the RPA Project Manager permissions must log out from RPA Manager and then log in again to appear in the list of available project managers. | W-13833036

Reviewer 2: ```suggestion
| A user with recently added RPA Project Manager permissions isn't available in the dropdown menu for project manager assignments. | The user with the RPA Project Manager permissions must log out from RPA Manager and then log in again to appear in the list of available project managers. | W-13833036
```
